content/posts/2022-features.md

```diff
@@ -0,0 +1,75 @@
+koji-build: as a reaction to a new dist-git commit, a new Koji build is triggered (you can specify allowed authors of a commit or merged pull request).
+bodhi-update: as a reaction to a successfully finished Koji build, a new Bodhi update is created
+
+TODO: imag (bodhi-update?)
+
+But that wasn’t all for this year. At the very end of the year, the Packit team implemented an alternative to the `propose-downstream` job that we call `pull-from-upstream`. The logic of the job is the very same: the source archive is saved to a lookaside cache, spec-file is updated and sent as a pull request to Fedora dist-git. The only – and main – difference is that the job is defined downstream (in the main dist-git branch) so you don’t need to install Packit in the upstream repository. The information about a new release is received from the [Upstream Release Monitoring](). The `pull-from-upstream` job is mainly targeted to the Fedora maintainers without upstream access or with upstream not being supported by Packit. (This job works with any upstream using GIT.) The setup is nicely described in [this blog post](). And if you want a dedicated documentation page for the Fedora downstream automation, look at https://packit.dev/docs/fedora-releases-guide/.
+
+TODO: image (pull-from-upstream)
+
+And that’s still not all, we’ve also added `propose-downstream` and `pull-from-upstream` to our dashboard. You can now see them in the Pipelines view. Also, logs can be checked on a detail page (that can be accessed from a status of a release commit).
+
+TODO: image (pipelines with propose-downstream>)
+
+Still not convinced we’ve done a lot? When using various downstream jobs, you can still hit an issue now and then. (Fedora infrastructure can forget to let us know or some flake error can occur.) Packit tries to retry automatically to overcome temporary issues, but sometimes it’s not enough or there is a real problem that needs to be fixed elsewhere. We can’t resolve the real problems for you, but we can help let you retry the job when needed. And you have two places where to do that.
+
+As you might be used to with the `propose-downstream` job, you can use comments in an upstream issue to retrigger Bodhi updates and Koji builds as well. Just configure [`issue_repository`](https://packit.dev/docs/configuration/#issue_repository) so Packit knows where to create issues in case of problems. (This does not need to be an upstream issue and this repository can be used for multiple projects.) Alternatively, if you use dist-git pull requests (either made by Packit or other maintainers), you can use a pull request comment to retrigger Koji Build or Bodhi update.
+
+TODO: image (issue to retrigger)
+
+## SRPM in Copr
+
+During the year, you might have realised that Packit has moved the builds of SRPM files to Copr. For more details, take a look at the [relevant page](), but let’s shortly describe the benefits:
```

Comment:
I would probably word this part differently? We didn't move SRPMs to Copr in '22, only introduced it and used it for new installations, so… something like…

> Throughout the 2022 we have implemented support for building SRPMs in the Copr and slowly started introducing newly-onboarded projects to use Copr right from the start. The definitive switch to Copr has been done at the beginning of 2023.

Reply:
I didn't want to mention all the details of the process but your version is much clearer. Thanks!